pipit/config.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022-2023 Parallel Software and Systems Group, University of
+# Maryland. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+
+# Private dict that stores global config values
+_config = {
```

Comment:
This looks good for a basic config manager. Here is a draft implementation in Hatchet: https://github.com/hatchet/hatchet/pull/511/files. It's similar to Pandas `set_options`. `registered_options` stores the default settings and `global_config` stores the current settings. The `get_option`, `set_option`, and `reset_option` functions are straightforward. There are also some validators to check if the given inputs are correct but I think the validator implementation has not finished yet.

Reply:
Done